dev/com.ibm.ws.security.saml.sso_fat.logout.ibm_security_logout/bnd.bnd

```diff
@@ -0,0 +1,38 @@
+#*******************************************************************************
+# Copyright (c) 2021 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src: \
+    fat/src
+
+fat.project: true
+
+tested.features: jsp-2.3, jdbc-4.1, el-3.0, servlet-4.0, samlweb-2.0, jdbc-4.0
```

Comment:
There are tests using jdbc-4.x in this bucket?

Reply:
I had to add those to satisfy the overall fat tooling (it spits out failures when it thinks features are not listed that are being used).